crates/fj/src/abi/model.rs

```diff
@@ -46,7 +46,11 @@ impl From<Box<dyn crate::models::Model>> for Model {
                 model.metadata()
             })) {
                 Ok(meta) => meta.into(),
-                Err(payload) => crate::abi::on_panic(payload),
+                Err(payload) => {
+                    model.metadata().name =
+                        crate::abi::on_panic(payload).into();
```

Comment:
I think we should adapt this method to return a `Result` and return an `Err` if a panic happens. Just parking the error message in the metadata, where it might not be noticed, is way too error-prone.